addons/mail/static/src/model/model_field_command.js

```diff
@@ -9,7 +9,7 @@ odoo.define('mail/static/src/model/model_field_command.js', function (require) {
 class FieldCommand {
     /**
      * @constructor
-     * @param {function} func function to call when executing this command
+     * @param {function} func function to call when executing this command. The function should ALWAYS return a boolean value to indicate if the field value is changed or not
```

Comment:
Just a remark: we follow max 80 char per line when it's possible.
We are less strict with actual code, but it's always possible and easy to keep it under 80 when writing comment.

Same in the commit message.

Reply:
fixed